src/query/Index.cpp

```diff
@@ -205,9 +204,8 @@ std::vector<TriangleMatch> Index::getClosestTriangles(const Eigen::VectorXd &sou
   std::vector<TriangleMatch> matches;
   rtree->query(bgi::nearest(sourceCoord, n),
                boost::make_function_output_iterator([&](TriangleTraits::IndexType const &match) {
-                 matches.emplace_back(bg::distance(sourceCoord, _mesh->triangles()[match.second]), match.second);
+                 matches.emplace_back(match.second);
                }));
-  std::sort(matches.begin(), matches.end());
```

Comment:
What happened with the sort?

Reply:
The XMatches are now only IDs, so sorting them doesn't make sense any longer.
Conceptually, the `getClosestX` now return sets.